docs/readthedocs/source/doc/LLM/Quickstart/install_linux_gpu.md

```diff
@@ -94,6 +94,28 @@ IPEX-LLM currently supports the Ubuntu 20.04 operating system and later, and sup
     <img src="https://llm-assets.readthedocs.io/en/latest/_images/gawk.png" width=100%; />
 
 
+#### For linux MTL
```

Comment:
1. it seems to be not suitable to add the section here. As linux MTL seems not be parallel with linux kernel 6.5.
    It could be a subsection of Linux kernel 6.5 or 6.2
2. MTL is not an official name which could be put into documentation: maybe "Intel Core Ultra iGPU" instead
3. Is this step only required for Core Ultra iGPU? How about other iGPU likes UHD/Iris?
4. We may also update the guide here: https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#linux